June 29 2023 Application Update (#2703)
SumoLogic/sumologic-documentation: blog-cse/2023-06-29-application-update.md

---
title: June 29, 2023 - Application Update
keywords:
  - cloud siem
  - rbac
  - related entities graph
  - detection time
tags: [application update]
authors:
  - url: https://help.sumologic.com/release-notes-cse/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

import useBaseUrl from '@docusaurus/useBaseUrl';

### New RBAC Capabilities

> Starting **Thursday, July 6**, a new RBAC capability will be introduced for Cloud SIEM Enterprise (CSE). They are *View Entities* and *Manage Entities*. Users with the built-in administrator role will received these capabilities automatically, but **admins must manually add these capabilities** to other roles as appropriate. If a user has neither role, they will not be able to see Entity details or interact with/manage Entities in any way.

Reviewer: ```suggestion
Starting **Thursday, July 6**, we're introducing new RBAC capabilities for Cloud SIEM Enterprise (CSE): *View Entities* and *Manage Entities*. Users with the built-in administrator role will receive these capabilities automatically, but **admins must manually add these capabilities** to other roles as appropriate. If a user does not have either role, they will not be able to see Entity details or interact with/manage Entities in any way.
```

### Minor Changes and Enhancements

* [New] Nodes can now be moved around individually on the Insight Related Entities Graph.

> * [Update] To align more closely with accepted industry definitions, we are changing the *Dwell Time* label on Insight metrics in the UI to *Detection Time*. Note that only the label is changing, not now the metric is calculated (i.e. the period of time between when the first record in an Insight was observed and when the Insight was created).

Reviewer: ```suggestion
* [Update] To align more closely with accepted industry definitions, we are changing the *Dwell Time* label on Insight metrics in the UI to *Detection Time*. Note that only the label is changing, not now the metric is calculated (i.e., the period of time between when the first record in an Insight was observed and when the Insight was created).
```
